man/systemd-debug-generator.xml

```diff
@@ -56,9 +56,10 @@
     <option>rd.systemd.debug_shell</option> option is
     specified, the debug shell service
     <literal>debug-shell.service</literal> is pulled into the boot
-    transaction. It will spawn a debug shell on tty9 during early
-    system startup. Note that the shell may also be turned on
-    persistently by enabling it with
+    transaction and a debug shell will be spawned during early boot.
+    By default, tty9 is used, but a specific tty can also be set, either with or without
```

Comment:
tty9 is configurable through `-Ddebug-tty=...`. Please export this setting in `man/custom-entities.ent.in` and use `&DEBUGTTY;` here.